app/services/builders/external_pull_request.rb

```diff
@@ -8,17 +8,27 @@ def initialize(pull_request_event_data, external_project)
     def call
       pull_request_payload = @pull_request_event_data.dig(:payload, :pull_request)
 
-      ::ExternalPullRequest
-        .find_or_create_by!(github_id: pull_request_payload[:id]) do |pull_request|
-        pull_request.html_url = pull_request_payload[:html_url]
-        pull_request.body = pull_request_payload[:body]
-        pull_request.title = pull_request_payload[:title]
-        pull_request.opened_at = pull_request_payload[:created_at]
-        pull_request.owner_id = User.find_by!(
+      pull_request = ::ExternalPullRequest
+                     .find_or_create_by!(github_id: pull_request_payload[:id]) do |ext_pull_request|
+        ext_pull_request.html_url = pull_request_payload[:html_url]
+        ext_pull_request.body = pull_request_payload[:body]
+        ext_pull_request.title = pull_request_payload[:title]
+        ext_pull_request.opened_at = pull_request_payload[:created_at]
+        ext_pull_request.owner_id = User.find_by!(
           login: @pull_request_event_data.dig(:actor, :login)
         ).id
-        pull_request.external_project_id = @external_project.id
+        ext_pull_request.external_project_id = @external_project.id
       end
+      pull_request.update!(state: assign_state(pull_request_payload))
```

Comment:
I'd add at least a test for this change

Reply:
Is there any reason why not adding specs to `spec/services/builders/external_pull_request_spec.rb`?
If the change is in that file, I find it best to have it tested in the own class spec file (which doesn't mean testing it where you did is wrong, in fact it's a good idea to have both specs).

Reply:
I messed up the files, will move this to `spec/services/builders/external_pull_request_spec.rb`. thanks